src/main/CommandHandler.java

```diff
@@ -0,0 +1,28 @@
+package main;
+
+public class CommandHandler {
+
+	private Bank bank;
+	private Validator validator;
+
+	public CommandHandler(final Bank bank) {
+		this.bank = bank;
+		this.validator = new Validator(this.bank);
+	}
+
+
+	public void processCommand(final String command) {
+		if (validator.isValidCommand(command)) {
+			String[] splitCommands = command.split(" ");
+			
+			if (splitCommands[0].equalsIgnoreCase("create")) {
+				this.bank.createAccount(splitCommands[1], splitCommands[2]);
```

Comment:
your command handler should not assume the caller will always give you a valid command. What if the caller only sent in the word "create" as a command, your code will fail with an ArrayOutOfBoundsException. Protect your code!